src/assets/scss/_variables.scss

```diff
@@ -15,6 +15,12 @@ $color-accent-primary: #bd006b;
 $color-accent-secondary: #8d0050;
 $color-page-background: #f7f7f7;
 
+// Media query breakpoints - copied from bootstrap
+$screen-xs-min: 480px;
+$screen-sm-min: 768px;
+$screen-md-min: 992px;
+$screen-lg-min: 1200px;
```

Comment:
some day we can import bootstrap sass file instead of its compiled css file

Reply:
I agree that that would be a useful thing. :+1: